odpc.client/src/features/publicatie/components/DocumentForm.vue

```diff
@@ -113,6 +114,11 @@ const model = computed<PublicatieDocument[]>({
   set: (value) => emit("update:documenten", value)
 });
 
+const invalidHandler = (event: Event) => {
```

Comment:
dit kunnen we denk ik globaal afhandelen voor alle input types met capture mode:
```vue
<main @invalid.capture="invalidHandler">
```

Reply:
besproken: we laten het even zo. mogelijk zijn er pitfalls aan event capturing